exchanges/ftx/ftx_test.go

```diff
@@ -226,10 +226,23 @@ func TestGetFuture(t *testing.T) {
 
 func TestGetFutureStats(t *testing.T) {
 	t.Parallel()
-	_, err := f.GetFutureStats(context.Background(), "BTC-PERP")
+	perp, err := f.GetFutureStats(context.Background(), "BTC-PERP")
 	if err != nil {
 		t.Error(err)
 	}
+
+	if perp.Greeks != nil {
+		t.Fatal("greeks should not be contained in perpetual")
+	}
+
+	f, err := f.GetFutureStats(context.Background(), "BTC-MOVE-2021Q4")
```

Comment:
`f` is the name of the FTX varaible, even though this is only a local var, please rn